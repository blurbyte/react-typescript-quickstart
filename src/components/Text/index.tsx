import styled from '@styles/styledComponents';
import { Colors } from '@styles/colors';

const Text = styled.p`
  font-family: Arial, Helvetica, sans-serif;
  color: ${Colors.darkGray};
  margin: 0.8rem 0;

  & strong {
    font-weight: 700;
  }
`;

export default Text;
