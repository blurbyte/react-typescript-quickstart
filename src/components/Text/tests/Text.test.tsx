import * as React from 'react';
import { shallow } from 'enzyme';

import Text from '../index';

describe('Text', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Text />);
    expect(wrapper).toMatchSnapshot();
  });
});
