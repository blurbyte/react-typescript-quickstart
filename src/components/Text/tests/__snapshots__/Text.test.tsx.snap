// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
.c0 {
  font-family: Helvetica,Arial,sans-serif;
  color: #444444;
  margin: 0.8rem 0;
}

.c0 strong {
  font-weight: 700;
}

<p
  className="c0"
/>
`;
